batch/noxfile_config.py

```diff
@@ -13,5 +13,5 @@
 #  limitations under the License.
 
 TEST_CONFIG_OVERRIDE = {
-    "gcloud_project_env": "BUILD_SPECIFIC_GCLOUD_PROJECT",
+    "gcloud_project_env": "GOOGLE_CLOUD_PROJECT",
```

Comment:
Please don't change this. Batch tests need to spin up VMs and those consume quota (external IP, CPUs etc). With all tests running in single project, this will cause unnecessary quota errors and/or slow down the test execution.